packaging/crosswalk-bin.spec

```diff
@@ -69,6 +69,7 @@ BuildRequires:  pkgconfig(haptic)
 BuildRequires:  pkgconfig(icu-i18n)
 BuildRequires:  pkgconfig(libdrm)
 BuildRequires:  pkgconfig(libexif)
+BuildRequires:  pkgconfig(libnotify)
```

Comment:
Argh! My apologies: I thought Tizen had libnotify packaged, but it turns out their `notification` package is not libnotify, but rather some Tizen-specific notification API :(
I guess this won't work on Tizen at the moment, sigh.